### PR TITLE
20084 e2e pay stripe

### PR DESCRIPTION
**Pull request by nnorbert**
E2E test payment with Stripe provider

### CODE REVIEW

File: apps/payments/e2e/specs/transactions/stipe.spec.ts
Comment (rgrnearform) [inline apps/payments/e2e/specs/transactions/stipe.spec.ts:1]: typo stipe.ts

**Review by rgrnearform — COMMENTED** [review]
typo

File: apps/payments/e2e/specs/transactions/stipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+test.describe("Transaction with stripe", () => {
+  const cards = {
+    success: "4242 4242 4242 4242",
+    genericDecline: "4000 0000 0000 0002",
+    insufficientFunds: "4000 0000 0000 9995",
+    incorrectNumber: "4242 4242 4242 4241",
+  };
+
+  const securityCode = "123";
+  let expirationDate: string;
```
Comment (peschina) [inline apps/payments/e2e/specs/transactions/stipe.spec.ts:26]: we could optionally just make it a const. Like a month in 2040 😄

File: apps/payments/e2e/specs/transactions/stipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    await stripe.fillCardNumber(cards.success);
+    await stripe.fillExpDate(expirationDate);
+    await stripe.fillSecurityCode(securityCode);
+    await stripe.pay();
+
+    await expect(
+      citizenPage.getByRole("img", { name: "Google" }),
+    ).toBeVisible();
+    await expect(citizenPage.url()).toContain(mockRedirectUrl);
+
```
Comment (peschina) [inline apps/payments/e2e/specs/transactions/stipe.spec.ts:73]: can we also check that the transaction status is correct in the payment request transaction list?

**Review by peschina — APPROVED** [review]
LGTM

File: apps/payments/e2e/specs/transactions/stipe.spec.ts (hunk 1 above)
Comment (nnorbert) [reply apps/payments/e2e/specs/transactions/stipe.spec.ts:26]: But then we will have to update it in every 20 years...  😄

Comment (peschina) [reply apps/payments/e2e/specs/transactions/stipe.spec.ts:26]: Why worry 😂  Jokes apart, I don't mind keeping it as it is

Comment (nnorbert) [reply apps/payments/e2e/specs/transactions/stipe.spec.ts]: I don't think that it is a big deal to calculate it dynamically, so I prefer this way honestly.

**Review by peschina — APPROVED** [review]

**Pull request merged**